### PR TITLE
Updating configs for Operator 1.2.0-beta

### PR DESCRIPTION
**Pull request by RakhithaRR**
### Purpose

- Updates MGW runtime and toolkit images

- Updates mgw related configs

- Updates k8s-artifacts related to APIM 3.2.0 and Analytics 3.2.0

### CODE REVIEW

File: api-operator/deploy/controller-configs/mgw_conf_mustache.yaml [hunk 1]
```diff
@@ -272,6 +274,56 @@ data:
       # username:password to create the connection to the central traffic manager
       throttleEndpointbase64Header = "admin:admin"
 
+      [throttlingConfig.binary]
+        enabled = true
```
Comment (pubudu538) [inline api-operator/deploy/controller-configs/mgw_conf_mustache.yaml:278]: This should be configurable. Only should work for scenario 12.

Comment (pubudu538) [reply api-operator/deploy/controller-configs/mgw_conf_mustache.yaml:278]: This is always enabled. So we dont need to do this change.

File: api-operator/deploy/controller-artifacts/operator.yaml [hunk 2]
```diff
@@ -33,7 +33,8 @@ spec:
       containers:
         - name: api-operator
           # Replace this with the built image name
-          image: wso2/k8s-api-operator:1.2.0-alpha
+#          image: wso2/k8s-api-operator:1.2.0-alpha
```
Comment (Ketharan) [inline api-operator/deploy/controller-artifacts/operator.yaml:36]: Any reason we keep this commented line? and referring non-wso2 image?

File: k8s-artifacts/api-portal/wso2apim-service.yaml [hunk 3]
```diff
@@ -39,4 +39,16 @@ spec:
...
+      targetPort: 5672
+      nodePort: 32005
+    - name: binary-receiver
+      port: 9611
+      targetPort: 9611
+      nodePort: 32006
+    - name: binary-auth
+      port: 9711
+      targetPort: 9711
+      nodePort: 32007
```
Comment (Ketharan) [inline k8s-artifacts/api-portal/wso2apim-service.yaml:54]: shall we have a new line here

File: api-operator/deploy/controller-artifacts/operator.yaml (hunk 2 above)
Comment (RakhithaRR) [reply api-operator/deploy/controller-artifacts/operator.yaml:36]: This was included only for pre-release testing purposes. Removed in latest code.

File: k8s-artifacts/api-portal/wso2apim-service.yaml (hunk 3 above)
Comment (RakhithaRR) [reply k8s-artifacts/api-portal/wso2apim-service.yaml:54]: Fixed with https://github.com/wso2/k8s-api-operator/pull/534

**Pull request merged**